gradle.properties

```diff
@@ -48,6 +48,7 @@ fabric-resource-conditions-api-v1-version=5.0.24
 fabric-resource-loader-v0-version=3.1.11
 fabric-screen-api-v1-version=2.1.0
 fabric-screen-handler-api-v1-version=1.3.136
+fabric-serialization-api-v1-version=1.0.0
```

Comment:
I think adding a new module is fine, I dont see any existing module that this would make sense for. I think its generic enough that more functionality can be added in the future. Im just making this comment to see if anyone else has thoughts on this as we are trying to be careful about the number of modules.